CMakeLists.txt

```diff
@@ -10830,13 +10831,6 @@ endif (gRPC_BUILD_TESTS)
 
 
 
-if (gRPC_INSTALL)
```

Comment:
I am not 100% sure what it was good for, but it was causing #8729 and based on some research I've done it should be fine to remove it. We can add it back if it turns out it's causing some issues.